my-app/src/Components/panicBtn/panicButton.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import './panicButton.scss'
+
+export default class PanicButton extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isClicked: false
+        };
+        this.onButtonClick = this.onButtonClick.bind(this);
+    }
+
+    onButtonClick() {
```

Comment:
read more about method syntax like a `onButtonClick = () => {....`
it is not required binding on `this`